mapstore/geostore-datasource-ovr.properties

```diff
@@ -0,0 +1,24 @@
+#Default Postgres Connection values 
+geostoreDataSource.driverClassName=org.postgresql.Driver
+geostoreDataSource.url=jdbc:postgresql://database:5432/georchestra
```

Comment:
Oops, had not seen this one : `20.0` branch uses `localhost` (non-docker context)